awslogs/core.py

```diff
@@ -295,3 +294,21 @@ def parse_datetime(self, datetime_text):
             date = date.replace(tzinfo=None)
 
         return int(total_seconds(date - datetime(1970, 1, 1))) * 1000
+
+    def process_json_message(self, message):
+        if message[0] == '{':
```

Comment:
How about this instead?

```
try:
    parsed = json.loads(message)
except ValueError:
    return message

if self.query is not None:
# ..... 
# .....
```

Assuming I guess the right exception, I like this better because checking the first character of the string looks odd to me. And this keeps the flow of the code tighter, i.e. the except/else is very close to the try/if.

Reply:
I agree + this enable jmespath query on logs containing arrays like 
`[{"a":"A value"}, {"a":"second A value", "b": "B value", "c":"C value"}]`

`--query='[].a'`
will produce : 
`["A value", "second A value"]`
or `--query='[[].a, [].c]'`
will produce : 
`[["A value", "second A value"], ["C value"]]`
or `--query='[].[a,c]'`
will produce : 
`[["A value", null], ["second A value", "C value"]]`